ui/src/components/VariantTables.js

```diff
@@ -3,7 +3,7 @@ import React, { useEffect, useMemo, useRef, useState } from 'react';
 import ReactTable from 'react-table';
 import { css } from '@emotion/react';
 
-import CustomPagination from '@arranger/components/dist/DataTable/Table/CustomPagination';
+// import CustomPagination from '@arranger/components/dist/DataTable/Table/CustomPagination';
```

Comment:
Usage of CustomPagination & EnhancedReactTable are updated in: https://github.com/nci-hcmi-catalog/portal/pull/1100